### PR TITLE
Update on Applying mobile specifc styles

### PR DESCRIPTION
**Pull request by david-ziffity**
## Purpose of this pull request

This pull request (PR) will update the section "Applying mobile specific styles"

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_mobile.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_mobile.md [hunk 1]
```diff
@@ -32,14 +33,21 @@ Use `styles-m.less` to generate mobile-specific styles.
...
     <css src="css/styles-m.css" />
   </head>
 </page>
 ```
 
-Media queries `@media-common`, `max screen__s` and `max` `screen__m` will be added to `styles-m.css`.
+Media queries `@media-common`, `max screen__m`, `max screen__s`, `max @screen__xs` and `max @screen__xxs`   will be added to `styles-m.css`.
 
-This will remove the desktop-specific files from your theme.
+Example of a Mixin to target screen width less than 480px:
```
Comment (rogyar) [inline src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_mobile.md]: Please, add a small clarification here

```suggestion
Example of a LESS mixin to target screen width less than 480px:
```

Thank you

Comment (david-ziffity) [reply src/guides/v2.3/frontend-dev-guide/responsive-web-design/rwd_mobile.md]: @rogyar Thank you.

**Comment by david-ziffity** [comment]
@dshevtsov Can you please update your status on this?

**Comment by dobooth** [comment]
running tests

**Pull request merged**